File: Tareas/Unidad2/a2253332351_practica02/13_IndiceMasaCorporal.cpp
#include <iostream>
using namespace std;

int main() {
    double peso;
    double estatura;
    double imc;

    cout << "Ingrese el peso en kilogramos: ";
    cin >> peso;
    cout << "Ingrese la estatura en metros: ";
    cin >> estatura;

    imc = peso / (estatura * estatura);

    cout << "El índice de masa corporal es: " << imc << endl;

    return 0;
}

File: Tareas/Unidad2/a2253332351_practica02/13_IndiceMasaCorporal.java
import java.util.Scanner;

public class IndiceMasaCorporal {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double peso, estatura, imc;

          System.out.println("Ingrese el peso en kilogramos: ");
          peso = entrada.nextDouble();
          System.out.println("Ingrese la estatura en metros: ");
          estatura = entrada.nextDouble();

          imc = peso / (estatura * estatura);

          System.out.println("El índice de masa corporal es: " + imc);
     }
}
